main/ble_server.c

```diff
@@ -327,21 +323,24 @@ void send_task(void *pvParameters)
                     free(event.buffer);
```

Comment:
hypothetically, would this work for really long messages? like can the bluetooth characteristic descriptor really handle being say... 0x100 bytes long? 0x1000?